### ISSUE TITLE
Add instructions on how to integrate into web application

### ISSUE THREAD

**Issue by mcintyre321** [dotnet/try#675]
I have always really wanted to make a git-backed wiki engine with c# powered pages, with a nice intellisense ready editor. It strikes me that this is the project to help with that.

Are there instructions on how I could integrate this project into a web app somewhere? I've had a look but haven't found anything which expressly guides me on how to do so.

**Comment by jonsequitur** [comment]
Which features are you thinking of? Local (`dotnet-try`) versus web-based (e.g. https://dotnet.microsoft.com/learn/dotnet/in-browser-tutorial/1) have a number of differences in capabilities.

**Comment by mcintyre321** [comment]
Are you talking about server-side execution vs webassembly? If so server side.

I think server-side execution, the idea would be for it to be edited by trusted users. Are the codeexecutions run inside a container, or under some other kind of sandbox? I

**Comment by jonsequitur** [comment]
Prior to our move to web assembly, server-side execution took place in transient containers because all users were anonymous. The infrastructure to manage these containers is not part of this repo. Web assembly offers improved scale, cost, and privacy, but limits you to netstandard2.0.

**Comment by mcintyre321** [comment]
And also limits you to web requests issued from the browser sandbox, right? Server side would allow for more powerful (and of course more dangerous) scripts.

**Comment by jonsequitur** [comment]
Right. The security boundary is the user's browser in the case of our web assembly implementation. In the server-based implementation, the security boundary is the machine with whatever limitations are placed on the account it runs under.

**Comment by mcintyre321** [comment]
It'd be great to have the option of runner. I'm more interested in the
server side model, however.
